packages/block-editor/src/store/private-selectors.js

```diff
@@ -175,13 +176,36 @@ export function getOpenedBlockSettingsMenu( state ) {
 /**
  * Returns all style overrides, intended to be merged with global editor styles.
  *
+ * Overrides are sorted to match the order of the blocks they relate to. This
+ * is useful to maintain correct CSS cascade order.
+ *
  * @param {Object} state Global application state.
  *
- * @return {Map} A map of style IDs to style overrides.
+ * @return {Array} An array of style ID to style override pairs.
```

Comment:
Don't we need to change the uses of `getStyleOverrides`?

Reply:
Good question. I don't believe so.

The only use I found of `getStyleOverrides` as a private selector, was in the `EditorStyles` component [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/editor-styles/index.js?plain=1#L78). The way the overrides are iterated over works the same whether the overrides are this array or the original Map.

I also tested this using this [change](https://github.com/WordPress/gutenberg/pull/57908/commits/453a981df4b7ab5e0b962d4e7c27216b1014f80b) on the primary [block style variations PR](https://github.com/WordPress/gutenberg/pull/57908) that originally updated the handling of style overrides.